app/src/offline/AndroidManifest.xml

```diff
@@ -0,0 +1,9 @@
+<manifest 
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <uses-permission 
+	android:name="android.permission.INTERNET"
+	tools:node="remove"/>
```

Comment:
Got the change here (after looking up http://tools.android.com/tech-docs/new-build-system/user-guide/manifest-merger#TOC-tools:attr-markers), but this means the offline version currently isn't offline, right?